tequilapi/contract/errcodes.go

```diff
@@ -140,4 +140,7 @@ const (
 	ErrorCodeProviderTransferredData = "err_provider_transferred_data"
 	ErrorCodeProviderSessionsCount   = "err_provider_sessions_count"
 	ErrorCodeProviderConsumersCount  = "err_provider_consumers_count"
+	ErrorCodeProviderEarningsSeries  = "err_provider_earnings_series"
+	ErrorCodeProviderSessionsSeries  = "err_provider_sessions_series"
+	ErrorCodeProviderTrafficSeries   = "err_provider_traffic_series"
```

Comment:
In some places, it called `data`, in other `traffic` or `transferred` or `TransferredData`. 
Please use the same naming everywhere.